packages/frontend/src/components/common/Filters/FieldAutoComplete.tsx

```diff
@@ -24,6 +27,14 @@ const AutocompleteMaxHeight = createGlobalStyle`
   }
 `;
 
+const getFieldIcon = (field: Item) => {
+    if (isField(field)) {
+        if (isDimension(field)) return getItemIconName(field.type);
+        else if (isMetric(field)) return getItemIconName(field.type);
+    }
```

Comment:
```suggestion
    if (isField(field) && (isDimension(field) || isMetric(field))) {
        return getItemIconName(field.type);
    }
```

Reply:
:D I did it before you suggested it https://github.com/lightdash/lightdash/pull/3486/commits/b16d19083164bfcc5ef6d529c3a319fbfbc87373